### PR TITLE
Nan-filled point data

### PR DESCRIPTION
**Pull request by emma-nechamkin**
This adds `FUDS_RAW` and `FUDS_ET` style columns for FUDS and AML as covered in #1861's discussion

### CODE REVIEW

File: data/data-pipeline/data_pipeline/content/config/csv.yml [hunk 1]
```diff
@@ -359,6 +359,12 @@ fields:
 - score_name: Is there at least one abandoned mine in this census tract?
   label: Is there at least one abandoned mine in this census tract?
   format: bool
+- score_name: Is there at least one Formerly Used Defense Site (FUDS) in the tract, where missing data is treated as False?
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/content/config/csv.yml:362]: added columns to downloadable

File: data/data-pipeline/data_pipeline/etl/score/constants.py [hunk 2]
```diff
@@ -309,8 +309,10 @@
     field_names.TRACT_PERCENT_NON_NATURAL_FIELD_NAME
     + field_names.PERCENTILE_FIELD_SUFFIX: "IS_PFS",
     field_names.NON_NATURAL_LOW_INCOME_FIELD_NAME: "IS_ET",
-    field_names.AML_BOOLEAN: "AML_ET",
-    field_names.ELIGIBLE_FUDS_BINARY_FIELD_NAME: "FUDS_ET",
+    field_names.AML_BOOLEAN: "AML_RAW",
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/etl/score/constants.py:312]: New colnames in tiles

File: data/data-pipeline/data_pipeline/etl/score/etl_score.py [hunk 3]
```diff
@@ -322,6 +322,8 @@ def _add_percentiles_to_df(
         # which are now deprecated.
         if not drop_tracts:
             # Create the "basic" percentile.
+            ## note: I believe this is less performant than if we made a bunch of these PFS columns
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/etl/score/etl_score.py:325]: Just a note for when we refactor

File: data/data-pipeline/data_pipeline/etl/score/etl_score.py [hunk 4]
```diff
@@ -538,9 +540,12 @@ def _prepare_initial_df(self) -> pd.DataFrame:
 
         df_copy[numeric_columns] = df_copy[numeric_columns].apply(pd.to_numeric)
 
-        # coerce all booleans to bools
+        # coerce all booleans to bools preserving nan character
+        # since this is a boolean, need to use `None`
```
Comment (emma-nechamkin) [inline data/data-pipeline/data_pipeline/etl/score/etl_score.py:544]: Oops! i was coercing these to bools from nulls, this will preserve

**Review by vim-usds — APPROVED** [review]
LGTM!

**Pull request merged**